index.html

```diff
@@ -36,7 +36,7 @@ <h2>UAV Log viewer</h2>
 
   <tr>
     <td>
-      <a href="./HardwareReport" style="margin: 20px; display: inline-block;"><img src="../images/HardwareReport_Icon.png" style="width:200px"></a>
+      <a href="./HardwareReport" style="margin: 20px; display: inline-block;"><img src="images/HardwareReport_Icon.png" style="width:200px"></a>
```

Comment:
The filetype is PNG (in caps) this server must be linux based I guess, I tested with this name the image is broken, but if I put in PNG it works.